docs/beginner/tutorial9-models/README.md

```diff
@@ -62,6 +62,8 @@ impl Vertex for ModelVertex {
 
 This is basically the same as the original `VertexBufferLayout`, but we added a `VertexAttribute` for the `normal`. Remove the `Vertex` struct in `lib.rs` as we won't need it anymore, and use our new `Vertex` from `model` for the `RenderPipeline`.
 
+We will also remove our homemade `vertex_buffer`, `index_buffer` and `num_indicdes`.
```

Comment:
There's a typo here: it should be `num_indices` not `num_indicdes`

Reply:
I fixed it! Thanks for review :)